src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2321,15 +2321,15 @@ Uninstall the following packages: {0}?</value>
     <comment>Label used in the general tab</comment>
   </data>
   <data name="PreferencesNodeAutocompleteMethod" xml:space="preserve">
-    <value>Default method</value>
+    <value>Default Ranking Method</value>
     <comment>Label used in the features tab</comment>
   </data>
   <data name="ObjectType" xml:space="preserve">
-    <value>Object type</value>
+    <value>Node Type Match</value>
     <comment>Label used in the features tab</comment>
   </data>
   <data name="MachineLearning" xml:space="preserve">
-    <value>ML Recommendation</value>
+    <value>Recommended Nodes</value>
```

Comment:
So the 2 options are: `Node Type Match` and `Recommended Nodes` in the preference panel?
@QilongTang @Jingyi-Wen 
Also, In case we are going with `Recommended Nodes` I think we should change the string from `MachineLearning` to something else

Reply:
Thinking about this more, in the future we are also going to show nodes based on usage. This will be under the same section where we show ML recommended nodes. So we should have a title that suits both the cases.